workloads/clearcontainers-3.x.yaml

```diff
@@ -110,47 +56,20 @@ runcmd:
  - echo "PATH=\"$PATH:/usr/local/go/bin:{{template "GOPATH" .}}/bin\""  >> /etc/environment
 
  - {{beginTask . "Downloading Go" }}
- - {{download . "https://storage.googleapis.com/golang/go1.8.3.linux-amd64.tar.gz" "/tmp/go1.8.3.linux-amd64.tar.gz"}}
+ - {{download . "https://dl.google.com/go/go1.9.2.linux-amd64.tar.gz" "/tmp/go1.9.2.linux-amd64.tar.gz"}}
```

Comment:
The latest 1.9.x release is 1.9.4. Is there a reason why .2 is chosen?

Reply:
CC is known to work with 1.9.2